### PR TITLE
Fixes #23.

### LINKED ISSUE #23: Confusion around API token
When setting up the project, I was initially confused while looking for the JIRA API token mentioned in the [README](https://github.com/mcurlej/joft?tab=readme-ov-file#usage). I eventually discovered that it accepts the Personal Access Token (PAT) instead, as outlined in the official JIRA [documentation](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html). Updating the documentation to clarify this would help avoid similar confusion.

### PR DESCRIPTION
**Pull request by mcurlej**
+ added unit tests for load_toml_app_config function

### CODE REVIEW

**Comment by mcurlej** [comment]
@frenzymadness wdyt?

File: tests/test_utils.py [hunk 1]
```diff
@@ -12,8 +18,69 @@ def test_load_valid_yaml() -> None:
...
+    """ 
+    Test that we will end with a non-zero error code when there is no config present and 
+    printing a message on the stdout.
+    """
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        mock_cwd.return_value = tmpdir
+
+        dir_names = ["etc", ".config"]
+        for name in dir_names:
```
Comment (frenzymadness) [inline tests/test_utils.py:72]: variable used only once
```suggestion
        for name in "etc", ".config":
```

File: tests/test_utils.py [hunk 2]
```diff
@@ -12,8 +18,69 @@ def test_load_valid_yaml() -> None:
...
+        mock_platformdirs.site_config_dir.return_value = ""
+
+        with unittest.mock.patch('sys.stdout', new= io.StringIO()) as mock_stdout:
+            with pytest.raises(SystemExit) as sys_exit:
+                joft.utils.load_toml_app_config()
+    
+    exit_code = 1
+    
+    assert "Cannot find configuration file" in mock_stdout.getvalue()
+    assert exit_code == sys_exit.value.args[0]
```
Comment (frenzymadness) [inline tests/test_utils.py:86]: Variable used only once
```suggestion
    
    assert "Cannot find configuration file" in mock_stdout.getvalue()
    assert  sys_exit.value.args[0] == 1
```

**Review by frenzymadness — COMMENTED** [review]
Most of the comments here are my personal preferences so you are free to ignore them if you want. It works so it's fine.

File: tests/test_utils.py [hunk 3]
```diff
@@ -12,8 +18,69 @@ def test_load_valid_yaml() -> None:
...
+
+        dir_names = ["etc", ".config"]
+        for name in dir_names:
+            config_dir = tmpdir + "/" + name
+            os.makedirs(config_dir)
+
+        mock_platformdirs.user_config_dir.return_value = tmpdir + "/" + ".config"
+        mock_platformdirs.site_config_dir.return_value = ""
+
+        with unittest.mock.patch('sys.stdout', new= io.StringIO()) as mock_stdout:
```
[the remark this answers is not in the document]
Comment (mcurlej) [reply tests/test_utils.py:79]: ack

File: tests/test_utils.py (hunk 1 above)
Comment (mcurlej) [reply tests/test_utils.py:72]: ack

File: tests/test_utils.py [hunk 4]
```diff
@@ -12,8 +18,69 @@ def test_load_valid_yaml() -> None:
...
+    hostname = "test"
+    pat_token = "pat_token"
+
+    config_file_contents = b"""[jira.server]
+    hostname = "test"
+    pat_token = "pat_token"
+    """
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        mock_cwd.return_value = tmpdir
```
[the remark this answers is not in the document]
Comment (mcurlej) [reply tests/test_utils.py:42]: the paths with / and spaces are really hard to read IMHO.

**Pull request merged**